Lombiq.Tests.UI/Services/WebDriverFactory.cs

```diff
@@ -167,4 +193,66 @@ private class StaticVersionEdgeConfig : EdgeConfig
     {
         public override string GetLatestVersion() => "83.0.478.37";
     }
+
+    private static async Task<string> TryFindVersionAsync(IDriverConfig driverConfig)
```

Comment:
The Windows logic for this is included in WebDriverManager.

Reply:
What part?

Reply:
WebDriverManager auto-selects the matching version for Chrome on Windows.

Reply:
Yeah, but it uses Regex which I don't trust. If chrome is in the path, my solution finds it with powershell, and if it's not it will still fall back to the MatchingBrowser thing from WebDriverManager

Reply:
Alright :D.

![image](https://user-images.githubusercontent.com/1976647/168443427-e52163d8-be70-44e5-b278-16960231746d.png)

Reply:
lol typo, meant to write regedit. I trust regex implicitly :D

Reply:
I did some testing and now I understand why you actually need regedit there. Apparently Windows now logs app paths in the registry even for apps that aren't in the `%PATH%`. How? No idea.